### PR TITLE
styra-link: update developer docs

### PR DESCRIPTION
**Pull request by msorens**
### What code changed, and why?

Coming back to this project with fresh eyes, I found a few various and sundry facts that could aid other code readers.

### Definition of done

Clarity++

### How to test

N/A

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,11 +7,5 @@
         "out": true // set this to false to include "out" folder in search results
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-    "typescript.tsc.autoDetect": "off",
-    "cSpell.words": [
-        "eopa",
-        "norc",
-        "picomatch",
-        "tsandall"
-    ]
```
Comment (msorens) [inline .vscode/settings.json:16]: Moved to cspell.json where the bulk of such things are.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,18 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Added
 
-- Added support for Enterprise OPA preview configuration
-- Added support for executing Enterprise OPA previews via code-lense, menus, and commands
-- Added Enterprise OPA specific output pane for preview results 
-- Added SUPPORT.md for more standardized repo organization
+- Added support for Enterprise OPA preview configuration.
+- Added support for executing Enterprise OPA previews via code-lense, menus, and commands.
+- Added Enterprise OPA specific output pane for preview results .
+- Added SUPPORT.md for more standardized repo organization.
```
Comment (msorens) [inline CHANGELOG.md:25]: Consistency with rest of file.

**Review by msorens — COMMENTED** [review]
Author notes below...

**Review by charlieegan3 — APPROVED** [review]
LGTM 👍

**Pull request merged**